website/docs/reference/appsmith-framework/context-object.md

```diff
@@ -52,6 +55,95 @@ This object contains all the values associated with the current URL that the use
 }
 ```
 
+#### host
+The host property of the URL interface is a string that includes the hostname, and then, a ```:```, followed by the port of the URL(if the [port](/reference/appsmith-framework/context-object#port) is available).
+
+
+Example:
+```js
+//{{appsmith.URL.host}}
+host:"app.appsmith.com:111"
+```
+
+#### hostName
+The hostname property of the URL is a **string** that holds the **domain of the URL**. 
+
+
+:::info
+hostname is the host name (without the port number).
+:::
+
+Example:
+```js
+//{{appsmith.URL.hostname}}
+hostName:"app.appsmith.com"
+```
+
+
+#### fullPath
+A full-path URL designates an EXACT location (such as a page, app, file, etc.). In addition to the domain and TLD, a full-path URL needs to include the protocol, subdomain (such as "app," "support," etc.), path/destination, and possibly a file extension as well as parameters. A full-path can include:
```

Comment:
Instead of TLD, please define the abbreviation - Top Level Domain (TLD), and then in the doc below refer to it as TLD abbreviation.

Reply:
Keeping the full form - Top Level Domain at both places